From 505d11da6cb8bd25eb0820fed78c69a9c4a867f5 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 12 May 2015 19:39:24 +0200
Subject: [PATCH] Disable pylint BR

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 85d82a7..a5cbed5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -218,7 +218,7 @@ if test "x$enable_python" != "xno"; then
 fi
 
 # Output the results of the Python checks
-if test "x$have_python" != "xyes" -o "x$have_pygobject" != "xyes" -o "x$PYLINT" = "xno"; then
+if test "x$have_python" != "xyes" -o "x$have_pygobject" != "xyes"; then
 	if test "x$enable_python" = "xyes"; then
 		AC_MSG_ERROR([python >= $PYTHON_REQS, $PYTHON_MODULES or pylint not found])
 	elif test "x$enable_python" = "xautodetect"; then
-- 
2.4.0

